Jenkinsfile

```diff
@@ -0,0 +1,27 @@
+pipeline {
+
+  agent any
+
+  stages {
+    stage("Protect Repo") {
+      steps {
+        withCredentials([[$class: 'UsernamePasswordMultiBinding', credentialsId:'github-diego', usernameVariable: 'USERNAME', passwordVariable: 'PASSWORD']]) {
```

Comment:
Suspeita de credencial encontrada, tipo: Typical Code Keywords

Se for um falso positivo, adicione o seguinte na mensagem do commit (amend) para que seja ignorada:
```
FalsoPositivo: b1e583ab
```